test/e2e-helm/e2e_helm_olm_test.go

```diff
@@ -77,17 +71,7 @@ var _ = Describe("Integrating Helm Projects with OLM", func() {
 			Expect(err).Should(Succeed())
 
 			By("generating the operator package manifests")
-			err = tc.Make("packagemanifests")
-			Expect(err).NotTo(HaveOccurred())
-
-			By("updating clusterserviceversion with the manager image")
-			testutils.ReplaceInFile(
-				filepath.Join(tc.Dir, "packagemanifests", operatorVersion,
-					fmt.Sprintf("e2e-%s.clusterserviceversion.yaml", tc.TestSuffix)),
-				"controller:latest", tc.ImageName)
-
-			By("installing crds to run packagemanifests")
-			err = tc.Make("install")
```

Comment:
+1

Reply:
OLM should be installing these in `run packagemanifests`.

Reply:
Yep!